_posts/2019-07-09-ranking-factors.md

```diff
@@ -45,20 +45,22 @@ Note: Sites using the search results API to present our results on their own web
 
 ### Core Ranking Algorithm
 
-Our system is built on Elasticsearch, which itself is built on Apache Lucene. For the first several generations, Elasticsearch used Lucene’s default ranking, the Practical Scoring Function. This Function starts with a basic Boolean match for single terms and adds in TF/IDF and a vector space model. Here are some high level definitions for these technical terms:
+Our system is built on Elasticsearch, which itself is built on Apache Lucene. 
+
+The Practical Scoring Function, the primary ranking algorithm, uses a basic Boolean match for single terms and adds in TF/IDF and a vector space model. Here are some high level definitions for these technical terms:
 
 * **Boolean matches** are the AND / OR / NOT matches you’ve probably heard about.
   * This AND that 
   * This OR that
   * This NOT that
   * This AND (that OR foo) NOT bar
-  * **Note** that while the relevance ranking takes these into account, we do not currently use these operators if entered by a searcher. Support for user-entered Boolean operators is coming in 2019.
+  * **Note:** While the relevance ranking takes these into account, we do not currently use these operators if entered by a searcher.
```

Comment:
Can we say that support for user-entered Boolean operators is on our roadmap, and and just leave the date off?